README.markdown

```diff
@@ -77,57 +85,63 @@ You can still add values in a string (or array) beyond the default, but you can
 
 By default, if a key includes additional values besides the ones you specify through this module, Puppet leaves those extra values in place. To change that, use the `purge_values => true` parameter of the `registry_key` resource. **Enabling this feature deletes any values in the key that are not managed by Puppet.**
 
-The `registry::purge_example` class provides a quick and easy way to see a demonstration of how this works. This example class has two modes of operation determined by the Facter fact `PURGE_EXAMPLE_MODE`: 'setup' and 'purge'.
+The `registry::purge_example` class provides a quick and easy way to see a demonstration of how this works. This example class has two modes of operation determined by the Facter fact `PURGE_EXAMPLE_MODE`: `setup` and `purge`.
```

Comment:
If setup and purge are strings (and I think they are), they should be in single quotes. If they aren't strings for some reason I don't know about, let me know of course!